core/src/main/java/io/grpc/util/AdvancedTlsX509KeyManager.java

```diff
@@ -141,6 +141,22 @@ public Closeable updateIdentityCredentialsFromFile(File keyFile, File certFile,
     };
   }
 
+  /**
+   * Loads the private key and certificate chains from the local file paths. The contents are only
+   * read at the construction time and won't be updated afterwards.
+   *
+   * @param keyFile  the file on disk holding the private key
+   * @param certFile  the file on disk holding the certificate chain
+   */
+  public void loadIdentityCredentialsFromFile(File keyFile, File certFile) throws IOException,
```

Comment:
Seems this should also be named with "update" in its name. Update vs load isn't a useful distinction, especially since updateIdentityCredentials() is also one-shot.